### DISCUSSION TITLE
Contribute package to GNU ELPA

### DISCUSSION THREAD

**Discussion by yuuyins** [overideal/perject#3] [Ideas]
Requesting Perject for inclusion on http://elpa.gnu.org/. It seems feasible since all dependencies are either on GNU ELPA already or are GNU Emacs built-in.

So it would "only" need following instructions on https://git.savannah.gnu.org/cgit/emacs/elpa.git/plain/README (which includes copyright assignment to the Free Software Foundation).

https://www.gnu.org/software/emacs/manual/html_node/emacs/Contributing.html could be helpful as well.

I am mostly asking this because perject is the best in its category and integrates well with the GNU Emacs built-ins, and could lead to further adoption by GNU Emacs users.

**Comment by overideal** [comment]
I am very happy to hear that perject is useful to you.
I think it would be nice to add this package to GNU Elpa. Unfortunately, I currently don't have the time to deal with the bureaucracy involved in this. If you have the time and motivation, you could initiate the process and I would provide the FSF with the Copyright assignment. Otherwise, I hope to eventually contribute it to GNU Elpa (or NonGNU Elpa) myself, but that has a low priority for me at the moment.
